web3/middleware/pythonic.py

```diff
@@ -1,6 +1,10 @@
 import codecs
 import operator
 
+from cytoolz.curried import (
```

Comment:
@pipermerriam It seems like this should either be 
```
from eth-utils.toolz import (
  curried
)
```
and then in the code we say `curried.keymap...` 
-or- 
we pull the logic from [this deleted file](https://github.com/ethereum/web3.py/pull/1317/files#diff-b022462abb470e6f70ffa82d41cd9641L1) into eth-utils. Do you have a preference?

Reply:
Yes, your example of curry-ing the function locally using the `eth-utils` import is correct.